content/01.abstract.md

```diff
@@ -1,3 +1,4 @@
 ## Abstract {.page_break_before}
 
-
+This manuscript is a template (aka "rootstock") for [Manubot](https://manubot.org/ "Manubot"), a tool for writing scholarly manuscripts.
+Use this template as a starting point for your manuscript by following the directions on the [repository homepage](https://github.com/manubot/rootstock "Manubot Rootstock").
```

Comment:
I think it's best to not modify `01.abstract.md` to avoid conflicts. You could put these two sentences at the beginning of `02.delete-me.md` before the first header so they'd appear as part of the abstract in the rendered version.

Reply:
@vincerubinetti this got marked as resolved, although it doesn't currently appear to be resolved, so I unresolved it.